opencti-platform/opencti-front/tests_e2e/settings/parameters.spec.ts

```diff
@@ -45,3 +47,13 @@ test('Check Logo replacement', async ({ page }) => {
   logoSrc = await page.getByRole('link', { name: 'logo' }).locator('img').getAttribute('src');
   expect(logoSrc).toContain('static/images/logo');
 });
+
+test('Check Hidden Entities', async ({ page }) => {
+  const leftBarPage = new LeftBarPage(page);
+  const settingPage = new SettingsPage(page);
+  await page.goto('/');
+  await page.getByTestId('ChevronRightIcon').click();
+  await leftBarPage.clickOnMenu('Settings', 'Parameters');
+  await settingPage.getHiddenEntityTypeSelect().click();
+  await expect(settingPage.getExternalReference()).toBeVisible();
+});
```

Comment:
I'm not a fan of this test. E2E tests have a high cost (~30sec in CI for a small test) and so their purpose is to validate a user scenario that is important we know it works. 

Here we are just validating the presence of an item in a list, if we really want to check those kind of things it should be done with unit test using `vite` but not in E2E.

Reply:
Imo just remove it

Reply:
Well that was a user scenario as it was a bug and noticed by a openCTI user himself. 
So I don't understand, do we or do we not want tests ? I think the team needs to be precise on when to do or not do tests.

Reply:
I'll open a discussion between the team to discuss about it